oxd-server/pom.xml

```diff
@@ -10,6 +10,9 @@
     <url>http://ox.gluu.org</url>
 
     <properties>
+        <postgresql.version>42.2.9</postgresql.version>
+        <mysql.version>8.0.19</mysql.version>
```

Comment:
We have removed MySQL and Postgres dependencies. 

If we load driver class during runtime using `Class.forName()` then we also need to provide the path of the driver-jar file. 

So we have added all jars in `lib` folder into classpath using `$LIB/*` (in `oxd-start.sh` and other sh files.). So if we add driver class into lib it will be added to classpath and can be used by oxd.

`$javaExe -Djava.net.preferIPv4Stack=true -cp $LIB/oxd-server.jar:$LIB/* org.gluu.oxd.server.OxdServerApplication server $CONF`